files/etc/apk/personal_autoupdate.conf

```diff
@@ -0,0 +1,25 @@
+#############################################################################################
+#	Filename:		files/etc/apk/personal_autoupdate.conf  								#
+#	Purpose:		Configuration file for apk-autoupdate that override the default configs	#
+#	Authors:		Giulio Coa <34110430+giulioc008@users.noreply.github.com>				#
+#	License:		This file is licensed under the MIT.									#
+#############################################################################################
+
+can_upgrade() {
+	case_match "${_packages_blacklist}" "$1" && return 1
+
+	case "$1" in
+		apk-tools)
+			_apk --quiet --no-cache --simulate upgrade --self-upgrade-only "$1"
+			;;
+		*)
+			_apk --quiet --no-cache --simulate add --upgrade "$1"
+			;;
+	esac
+
+	return 0
+}
+
+finalize() {
+	:
+}
```

Comment:
Can you explain how this works? When does it run? Isn't `--simulate` not actually upgrading?

Reply:
https://github.com/jirutka/apk-autoupdate
https://github.com/jirutka/apk-autoupdate/blob/master/man/autoupdate.conf.5.adoc

I have overridden the default functions of this script https://github.com/jirutka/apk-autoupdate/blob/master/etc/autoupdate.conf